plugins/github-actions/package.json

```diff
@@ -28,6 +28,8 @@
     "@material-ui/core": "^4.9.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "@octokit/rest": "^18.0.0",
```

Comment:
If we are passing the octokit client in the constructor, do we still need is in this package as a dependency?

Reply:
We do need it as long as we want octokit parameter to be typed.